src/haddock/modules/analysis/alascan/defaults.yaml

```diff
@@ -0,0 +1,52 @@
+resdic_:
+  default: []
+  type: list
+  minitems: 0
+  maxitems: 100
+  title: List of residues to be mutated
+  short: List of residues to be mutated. By default all the interface residues are
+    mutated.
+  long: List of residues to be mutated. By default all the interface residues are
+    mutated. resdic_* is an expandable parameter. You can provide resdic_A,
+    resdic_B, resdic_C, etc, where the last capital letter is the chain
+    identifier.
+  group: analysis
+  explevel: easy
```

Comment:
Should this (and the other) be easy?

Reply:
it's an analysis module..I would give users the freedom to experiment. maybe only this resdic_ should be `expert`